erts/doc/src/erlang.xml

```diff
@@ -5278,13 +5278,22 @@ receive_replies(ReqId, N, Acc) ->
               programs with spaces in filenames or directory names.
               If spaces in executable filenames are desired, use
               <c>{spawn_executable, <anno>Command</anno>}</c> instead.</p>
-          </item>
-          <tag><c>{spawn_driver, <anno>Command</anno>}</c></tag>
-          <item>
-            <p>Works like <c>{spawn, <anno>Command</anno>}</c>, but demands
-              the first (space-separated) token of the command to be the name
-              of a loaded driver. If no driver with that name is loaded, a
-              <c>badarg</c> error is raised.</p>
+            <warning>
+              <p>On Unix systems, arguments are passed to a new operating
+                system process as an array of strings but on Windows it is up
+                to the child process to parse them and some Windows programs
+                may apply their own rules, which are inconsistent with the
+                standard C runtime <c>argv</c> parsing.</p>
+              <p>This is particularly troublesome when invoking <c>.bat</c> or
+                <c>.com</c> files as these run implicitly through
```

Comment:
`.com` files are binaries, aren't they (though probably not supported on modern systems at all). Did you mean `.cmd`...?

Reply:
My understanding is that `.com` are also problematic because they are handled by `cmd.exe` and invoked pretty much the same way as `.bat`.  :(

Reply:
I'll include all three, thanks. :)